File: server/src/sti_std/sti_std.ts
import * as admin from 'firebase-admin';
export class StiStd {
  name: string;
  genitalia: string;
  information: string;
  duration: number;

  constructor(name: string, genitalia: string, information: string, duration: number) {
    this.name = name;
    this.genitalia = genitalia;
    this.information = information;
    this.duration = duration;
  }
}

File: app/lib/models/sti_std.dart
import 'package:freezed_annotation/freezed_annotation.dart';

part 'sti_std.freezed.dart';
part 'sti_std.g.dart';

@freezed
class StiStd with _$StiStd {
  factory StiStd({
    required String name,
    required List<String> genitalia,
    required String information,
    required int duration,
  }) = _StiStd;

  StiStd._();

  factory StiStd.fromJson(Map<String, dynamic> json) => _$StiStdFromJson(json);
}
